helm/check-metric/values.yaml

```diff
@@ -0,0 +1,111 @@
+# Default values for check-metric.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: registry.cern.ch/cmsweb/check-metric
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "20220404-stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+args:
+  dbs: 
+  - /data/check-metric
+  - -url
+  - "http://prometheus-service.monitoring.svc.cluster.local:8080"
+  - -metric
+  - "dbsmigration_process_open_fds,dbs_migrate_exporter_process_open_fds,dbs_global_exporter_process_open_fds,dbs_globalW_exporter_process_open_fds,dbs_phys03R_exporter_process_open_fds,dbs_phys03W_exporter_process_open_fds"
```

Comment:
This should be fixed since  we no longer use dbs metrics since we decommissioned dbs services and use instead dbs2go. And, for dbs2go we do not have so far any outages, therefore i doubt we need it. As such I doubt we need dbs section here per-se.